backend/app/repositories/__init__.py

```
@@ -0,0 +1,7 @@
+"""Repositories package initialization."""
+
+from app.repositories.base import BaseRepository
+from app.repositories.item import ItemRepository
+from app.repositories.user import UserRepository
+
+__all__ = ["BaseRepository", "UserRepository", "ItemRepository"]
```

Ruff (0.12.2)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)